src/SFA.DAS.AssessorService.Application.Api/Services/OrganisationMergingService.cs

```diff
@@ -0,0 +1,280 @@
+
+            // Read all the standards from the secondary organisation.
+
+            foreach(var secondaryOrganisationStandard in secondaryOrganisation.OrganisationStandards)
+            {
+                // Does the primary organisation have this standard already?
+                var primaryOrganisationStandard = primaryOrganisation.OrganisationStandards.FirstOrDefault(pos => pos.StandardCode == secondaryOrganisationStandard.StandardCode);
+
+                if (null == primaryOrganisationStandard)
+                {
+                    // No - so add the standard to the primary organisation.
+
+                    primaryOrganisationStandard = new OrganisationStandard()
+                    {
+                        DateStandardApprovedOnRegister = secondaryOrganisationStandard.DateStandardApprovedOnRegister,
+                        OrganisationStandardData = secondaryOrganisationStandard.OrganisationStandardData,
+                        Comments = $"{secondaryOrganisationStandard.Comments} ** this standard has been merged from Organisation {secondaryOrganisation.EndPointAssessorOrganisationId}",
+                        ContactId = primaryContact?.Id,
+                        EffectiveFrom = secondaryOrganisationStandard.EffectiveFrom,
+                        EffectiveTo = secondaryOrganisationStandard.EffectiveTo,
+                        EndPointAssessorOrganisationId = primaryOrganisation.EndPointAssessorOrganisationId,
+                        StandardCode = secondaryOrganisationStandard.StandardCode,
+                        StandardReference = secondaryOrganisationStandard.StandardReference,
+                        Status = secondaryOrganisationStandard.Status,
+
+                        OrganisationStandardVersions = new List<OrganisationStandardVersion>(),
+                        OrganisationStandardDeliveryAreas = new List<OrganisationStandardDeliveryArea>(),
+                    };
+                    primaryOrganisation.OrganisationStandards.Add(primaryOrganisationStandard);
+                }
+
+                // Now read all the versions for this standard for the secondary organisation.
+                var secondaryOrganisationStandardVersions = _dbContext.OrganisationStandardVersion.Where(sosv => sosv.OrganisationStandardId == secondaryOrganisationStandard.Id);
+                foreach(var secondaryOrganisationStandardVersion in secondaryOrganisationStandardVersions)
+                {
+                    // Does the standard version exist for this standard in the primary organisation?
+
+                    var primaryOrganisationStandardVersion = primaryOrganisationStandard.OrganisationStandardVersions.FirstOrDefault(posv => posv.StandardUId == secondaryOrganisationStandardVersion.StandardUId && posv.Version == secondaryOrganisationStandardVersion.Version);
```

Comment:
StandardUID should be enough, it's a concatenation of IFAte ref + version -> ST0016_1.0